wordpress-mobile/WordPress-iOS
# Create a WordPressKit framework to house all remote services

Issue #7144

This PR creates a new framework called `WordPressKit` that will contain all of the remote service currently in the main project and the stats framework. This PR is the first step to starting to move things in - it creates the framework and adds it to the build.

Note that this PR is targeted at a feature branch, not develop. I'll be splitting the actions up into much smaller PRs following this pattern.

## To Test
1. `pod install`.
2. Build the WordPress target.
3. Build the WordPressKit target.
4. Make sure nothing breaks badly.

Needs review: @bummytime, @elibud
